specification/web/resource-manager/Microsoft.Web/stable/2022-09-01/WebApps.json

```diff
@@ -21567,6 +21567,415 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/deployWorkflowArtifacts": {
+      "post": {
+        "tags": [
+          "WebApps"
+        ],
+        "summary": "Creates the artifacts for web site, or a deployment slot.",
+        "description": "Description for Creates the artifacts for web site, or a deployment slot.",
+        "operationId": "WebApps_DeployWorkflowArtifacts",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/siteNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Artifacts deployed."
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Deploys workflow artifacts": {
+            "$ref": "./examples/PostDeployWorkflowArtifacts.json"
+          },
+          "Delete workflow artifacts": {
+            "$ref": "./examples/DeleteDeployWorkflowArtifacts.json"
+          }
+        },
+        "x-ms-long-running-operation": true
```

Comment:
Long running POST actions should return 202 with locaiton header  [https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/async-api-reference.md#call-action-post-asynchronously](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/async-api-reference.md#call-action-post-asynchronously)

Reply:
Updated the tag to reflect that this is not a long running operation